src/components/Table.jsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+import { DataGrid } from "@mui/x-data-grid";
+import style from "./Table.module.css";
+class DataGridDemo extends React.Component {
+  render() {
+    return (
+      <div style={{ height: 300, width: "90%" }}>
+        <DataGrid
+          className={style.tableBackground}
+          rows={this.props.valueRows}
```

Comment:
you can call this just rows instead of valueRows and after you will change the component to functional one, the this part won't be needed